flaskr/api/views.py

```diff
@@ -0,0 +1,31 @@
+from flask import abort, jsonify, current_app
+import json
+from . import app
+
+
+"""
+Return json for all courses under <spec> specialisation
+in <sem> semester (X1|S1|S2)
+"""
+@app.route('/specialisation/<spec>/<sem>')
+def specialisation(spec, sem):
+    sem = sem.upper()  # accept lower case request
+
+    db_file = "%s/%s" % (app.root_path, current_app.config['CLASSUTIL_PATH'])
```

Comment:
I'd suggest you use `current_app` for both of these for the sake of portability.